src/lib/components/tooltip/QTooltip.svelte

```diff
@@ -1,18 +1,12 @@
 <script lang="ts">
-  import { createClasses } from "$lib/utils";
   import type { QTooltipProps } from "./props";
 
   export let value: QTooltipProps["value"] = undefined,
     position: QTooltipProps["position"] = "bottom",
     userClasses: QTooltipProps["userClasses"] = "";
   export { userClasses as class };
-
-  $: classes = createClasses([value && "active", position || "top"], {
```

Comment:
The `|| "top"` seems to not have been added in the refactor, but it might be okay, since the prop defaults to `bottom` anyway.